cf-keycrypt/cf-keycrypt.c

```diff
@@ -0,0 +1,388 @@
+/*
+   cf-keycrypt.c
+
+   Copyright (C) 2017 cfengineers.net
```

Comment:
@kacf given that we are making significant changes to it, I suggest keeping the initial license intact, but prepend our license and copyright for 2017. Sounds right?